person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(name, age = 'unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] Your parameters arrangement here is incorrect. `name` should be initialized to "unknown", not `age`

- [ ] This `@id` here should be provided. It would suggest you auto-generate randomly.

  ```
  def initialize(age, name = 'unknown', parent_permission: true)
    @id =  rand(1...1000)
      ...
  ```

Reply:
well resolved! ✔️